src/widget/input_box.c

```diff
@@ -22,7 +22,10 @@ void input_box_start(input_box *box)
 {
     int text_width = (box->width_blocks - 2) * BLOCK_SIZE - 35;
     keyboard_start_capture(box->text, box->text_length, box->allow_punctuation, text_width, box->font);
-    clear_text_button.x_offset = box->x + box->width_blocks * 16 - 43;
+    clear_text_button.x_offset = box->x + box->width_blocks * 16;
+    if (!box->put_clear_button_outside_box) {
+        clear_text_button.x_offset -= 43;
```

Comment:
I know it wasn't originally like this, but can you use a `#define CLEAR_BUTTON_WIDTH 43` instead of having the number here?